pennylane_lightning_gpu/src/algorithms/ObsDatum.hpp

```diff
@@ -0,0 +1,78 @@
+#pragma once
```

Comment:
Do these need to be in separate files? I have no problem with it, as we will likely replace it for the new Lightning structure later, but just wondering about your thoughts on this.

Reply:
Nope! this is part of my previous work... I merge it with `cuOpsData` for now. Thanks.